app/controllers/carts_controller.rb

```diff
@@ -12,6 +12,12 @@ def create
     redirect_back(fallback_location: items_path)
   end
 
+  def update
+    item = Item.find(params[:item_id])
+    @cart.update_item(item.id, params[:Quantity])
```

Comment:
Nice. I think we will need to add to this so that we have a way of dealing with situations where the user changes the amount down to 0. Should be able to add some methods to the cart poro and a conditional to catch quantities of 0 to remove the item.